src/hooks/useChatCommand.ts

```diff
@@ -83,7 +86,12 @@ export default function useChatCommand(callback: (command: string) => void) {
         `*Twitch extension received command: ${commandName} (${command})*`,
         id,
       );
-      if (command) callback(command);
+      if (command) {
+        const isPrivileged = privilegedUsers.includes(
+          tags.username?.toLowerCase() ?? "",
+        );
+        callback(command, isPrivileged);
+      }
```

Comment:
I think we should probably handle the permissions check within this hook, rather than delegating this specific bit of permissions checking to the logic using this hook (we already do the mod checking here, so it seems odd to pass half the permissions checking on)?

To achieve that, either can hard-code the `!refresh` case, or it might be worth updating `commandsMap` to have values that have the canonical command name + a permissions level?